services/scan/src/scanners/fujitsu.ts

```diff
@@ -128,7 +128,7 @@ export class FujitsuScanner implements Scanner {
       scannedFiles.push(path);
       if (scannedFiles.length % 2 === 0) {
         results.resolve(
-          Promise.resolve(scannedFiles.slice(-2) as SheetOf<string>)
+          Promise.resolve(scannedFiles.slice(-2) as unknown as SheetOf<string>)
```

Comment:
I ran into this issue locally when I tried it out. What I'd probably do instead is something like this:

```ts
    let frontPagePath: string | undefined;
    let backPagePath: string | undefined;
    new StreamLines(scanimage.stdout).on('line', (line: string) => {
      const path = line.trim();
      void this.logger.log(
        LogEventId.FujitsuScanImageScanned,
        'system',
        {
          message: `scanimage [pid=${scanimage.pid}] reported a scanned file: ${path}`,
          disposition: 'success',
        },
        debug
      );

      if (!frontPagePath) {
        frontPagePath = path;
      } else if (!backPagePath) {
        backPagePath = path;
      } else {
        results.resolve(Promise.resolve([frontPagePath, backPagePath]));
        frontPagePath = undefined;
        backPagePath = undefined;
      }
    });
```

Reply:
Ended up solving like this:

```ts
      scannedFiles.push(path);
      if (scannedFiles.length % 2 === 0) {
        const [frontPath, backPath] = scannedFiles.slice(-2);
        results.resolve(Promise.resolve([frontPath, backPath]));
      }
```